### PR TITLE
Interfaceimp

### PR DESCRIPTION
**Pull request by vmhta**

### CODE REVIEW

File: AutofacSolution/Autofac.Entity/Class1.cs [hunk 1]
```diff
@@ -6,7 +6,19 @@
 
 namespace Autofac.Entity
 {
-    public class Class1
+    public static class DbOperations
```
Comment (asbhosale) [inline AutofacSolution/Autofac.Entity/Class1.cs]: Please use naming conventions. Class1.cs should also be renamed to DbOperations.cs

File: AutofacSolution/Autofac.SqlLogConsole/Program.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
+﻿using Autofac.Interface;
+using Autofac.Interface.Interface;
+using System;
+using System.Collections.Generic;
```
Comment (asbhosale) [inline AutofacSolution/Autofac.SqlLogConsole/Program.cs]: Please remove unused references.

File: AutofacSolution/Autofac.Entity/Class1.cs (hunk 1 above)
Comment (vmhta) [inline AutofacSolution/Autofac.Entity/Class1.cs]: the changes are made

File: AutofacSolution/Autofac.SqlLogConsole/Program.cs (hunk 2 above)
Comment (vmhta) [inline AutofacSolution/Autofac.SqlLogConsole/Program.cs]: Removed from all files

**Comment by asbhosale** [comment]
:shipit:

**Pull request merged**